index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Microverse project</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <main id="main">
+    <section id="title">
+      <h1>Jose Cruz Y Alejandro Velasquez - Tribute Website</h1>
+    </section>
+    <div id="img-div">
+      <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+        alt="">
+      <div id="tribute-information">
+        <div id="img-caption">
+          <h2>Irure labore eu voluptate sunt. Deserunt cupidatat fugiat dolor reprehenderit irure voluptate amet.</h2>
+        </div>
+        <div id="tribute-info">
+          <p>Laborum cillum dolore eiusmod duis. Anim consectetur exercitation incididunt aliqua exercitation occaecat.
+            Reprehenderit deserunt ipsum incididunt consequat consectetur. Culpa magna fugiat nisi nisi ipsum ea anim
+            nulla nostrud sunt elit. Dolore excepteur enim anim dolor voluptate exercitation quis est reprehenderit
+            aliqua aliquip elit.</p>
+          <p>Cupidatat deserunt sunt ea do non ipsum ex exercitation ea cillum sunt. Adipisicing consectetur
+            exercitation esse eiusmod dolor quis reprehenderit dolore est eu occaecat amet magna fugiat. Dolore non
+            cillum aliquip non sint laboris do minim deserunt eiusmod excepteur velit. Sint ea amet ullamco officia
+            fugiat laborum ad amet cillum ullamco voluptate ullamco elit.</p>
+
+          <div id="link-container">
+            <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">
+              <div id="button">
+                <p>Wikipedia Information</p>
+              </div>
+            </a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <footer>
+      <div id="footer-container">
+        <a href="https://github.com/josscodee">&lt Joss GitHub</a>
+        <p class="footer-text">- Microverse - 2023 -</p>
+        <a href="https://github.com/VelzckC0D3">Velzck GitHub ></a>
+      </div>
```

Comment:
This should be listed on a `<li>` inside of a `<ul>` according to [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)